MediaWikiKit/MediaWikiKit/MWKSectionList.m

```diff
@@ -140,4 +140,37 @@ - (NSArray*)entries {
     return self.sections;
 }
 
+#pragma mark - Hierarchy
+
+- (void)buildSectionHierarchy {
```

Comment:
Can/Should this be invoked automatically when the sections are set? (Rather than invoking it externally by the article VC)